src/components/pages/document/list/view/list/image.tsx

```diff
@@ -0,0 +1,84 @@
+import { Avatar, Flex, Link, Text } from "@chakra-ui/react";
```

Comment:
AFAIK this component is completely unnecessary since it's not imported anywhere, please ensure housekeeping once done with feature